### DISCUSSION TITLE
Why can't I use dde.icbc.PointSetOperatorBC

### DISCUSSION THREAD

**Discussion by ZHSLZ** [lululxvi/deepxde#1063] [Q&A]
Dear Dr. Lu Lu, 
I encountered the problem that the deepxde.icbc.boundary_conditions.PointSetOperatorBC function cannot be used when using deepxde. I hope you can tell me how to use the PointSetOperatorBC function correctly.
![image](https://user-images.githubusercontent.com/116650342/205443375-c767c0a6-81f1-4ef2-96f3-e6cbd096ba34.png)
![image](https://user-images.githubusercontent.com/116650342/205443400-3d86b17c-fb75-4548-8c85-4f4359557d7a.png)

**Comment by lululxvi** [comment]
It is not in pip or conda yet. You need to download the source code from Github.

**Reply by ZHSLZ** [reply]
Thank you very much. Through your guidance, I have successfully run it!

**Reply by lululxvi** [reply]
It is now in the new version v1.7.2
